CommonTools/PileupAlgos/plugins/PuppiProducer.cc

```diff
@@ -157,13 +160,15 @@ void PuppiProducer::produce(edm::Event& iEvent, const edm::EventSetup& iSetup) {
           }
           if (tmpFromPV == 1 || tmpFromPV == 2) {
```

Comment:
```else if``` 
(same at line 158)
(same at line 185)
(quite likely also same at lines 189 and 192)

Reply:
done